fedimint-api-client/src/api/global_api/with_cache.rs

```diff
@@ -562,4 +563,13 @@ where
         )
         .await
     }
+
+    async fn get_invite_code(&self, guardian: PeerId) -> PeerResult<InviteCode> {
+        self.request_single_peer(
+            INVITE_CODE_ENDPOINT.to_owned(),
+            ApiRequestErased::default(),
+            guardian,
+        )
+        .await
+    }
```

Comment:
Do you already have a fully initialized client when using this fn or are you somehow only initializing the api client? If you already have a client no API calls are necessary: https://docs.fedimint.org/fedimint_client/struct.Client.html#method.invite_code